test/parallel/test-fs-write-stream-partial-write.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const tmpdir = require('../common/tmpdir');
+const file = path.join(tmpdir.path, 'out.txt');
+
+tmpdir.refresh();
+
+{
+  const _fs_write = fs.write;
+  let i = 0;
+  fs.write = function(fd, data, offset, length, pos, cb) {
+    if (i === 0) {
+      const half = Math.floor(length / 2);
+      _fs_write(fd, data, offset, half, pos, cb);
+    } else {
+      assert(
+        offset > 0, 'second call to fs.write() must provide non-zero offset');
```

Comment:
Can you make the error message include the actual value of `offset`, e.g. using a template string?

Reply:
done, thanks!